### PR TITLE
BAEL-7351 Demo code

### PR DESCRIPTION
**Pull request by s9m33r**

### CODE REVIEW

**Comment by s9m33r** [comment]
The tests I've added are passing now on my local. However, I still see other tests failing on my local when I run all tests in spring-kafka-2 module. Can you please advice?

**Comment by s9m33r** [comment]
@asif-anwar we're good here right?

File: spring-kafka-2/src/test/java/com/baeldung/spring/kafka/groupId/MainLiveTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        consumer.resetLatch();
+    }
+
+    @Test
+    public void givenEmbeddedKafkaBroker_whenSendingWithSimpleProducer_thenMessageReceived() throws Exception {
+        String data = "Test 123...";
+
+        producer.send(data);
+
+        boolean messageConsumed = consumer.getLatch()
```
Comment (acerasoni) [inline spring-kafka-2/src/test/java/com/baeldung/spring/kafka/groupId/MainLiveTest.java:39]: We don't need a newline here

Comment (s9m33r) [reply spring-kafka-2/src/test/java/com/baeldung/spring/kafka/groupId/MainLiveTest.java:37]: Done.

File: pom.xml [hunk 2]
```diff
@@ -689,8 +689,8 @@
                 <module>apache-httpclient-2</module>
                 <module>apache-httpclient4</module>
                 <module>apache-httpclient</module>
-                <module>apache-kafka-2</module>
                 <module>apache-kafka</module>
+                <module>apache-kafka-2</module>
```
Comment (KevinGilmore) [inline pom.xml:693]: Is this change necessary for the topic at hand? Generally, we should avoid changes of this nature and leave them for the editorial team to clean up at a later date.

File: spring-kafka-2/src/main/java/com/baeldung/spring/kafka/groupId/MyKafkaConsumer.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package com.baeldung.spring.kafka.groupId;
```
Comment (KevinGilmore) [inline spring-kafka-2/src/main/java/com/baeldung/spring/kafka/groupId/MyKafkaConsumer.java:1]: Note that this module is already associated with 7 articles, which is our max for Spring modules. Let's move the code to the spring-kafka-3 module.

File: pom.xml (hunk 2 above)
Comment (s9m33r) [reply pom.xml]: It's not reverted.

File: spring-kafka-2/src/main/java/com/baeldung/spring/kafka/groupId/MyKafkaConsumer.java (hunk 3 above)
Comment (s9m33r) [reply spring-kafka-2/src/main/java/com/baeldung/spring/kafka/groupId/MyKafkaConsumer.java]: Done. The code has been moved to spring-kafka-3.

**Pull request merged**